backend/src/middleware/validate-Roles.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function HasRole(roles) {
```

Comment:
Lets just use camelCase here for the function name and the file name so that they are both 'validateRoles'. And also add an eslint rule for camelcase function names please! ;)

Reply:
I already make 'off' the camelCase inside eslint file, do you mean to make it 'on' ?

Reply:
Yes, its a good practice to enforce rules that help readability and match the JS style you will see in most projects. This way everyone will get used to using the same naming conventions.